src/tlapm_lib.ml

```diff
@@ -650,4 +650,14 @@ let module_of_string ~(content : string) ~(filename : string) ~loader_paths ~pre
          | Some l, None -> Error (Some l, Printexc.to_string e)
          | None, None -> Error (None, Printexc.to_string e))
 
+let module_of_string module_str =
+    let hparse = Tla_parser.P.use M_parser.parse in
+    let (flex, _) = Alexer.lex_string module_str in
+    Tla_parser.P.run hparse ~init:Tla_parser.init ~source:flex
+
 let stdlib_search_paths = Params.stdlib_search_paths
+
+let parse_module_of_string module_str =
```

Comment:
Is `parse_module_of_string` left here accidentally? It looks the same as `module_of_string`.

Reply:
Cleaned up things after the merge.